src/SDKs/Azure.ApplicationModel.Configuration/data-plane/Azure.ApplicationModel.Configuration/ConfigurationClient.cs

```diff
@@ -12,6 +12,9 @@
 
 namespace Azure.ApplicationModel.Configuration
 {
+    /// <summary>
+    /// ConfigurationClient can be used for all operations with a <see cref="ConfigurationSetting"/> in the Azure Configuration Store.
```

Comment:
Maybe we should consider something like `The client to use for interacting with the Azure Configuration Store` or something similar?   The name of the class leading the sentence feels a bit awkward to me.